src/riak_ql_parser.yrl

```diff
@@ -191,6 +198,7 @@ Funcall -> Identifier left_paren FunArg         right_paren : make_funcall('$1',
 Funcall -> Identifier left_paren asterisk       right_paren : make_funcall('$1', ['$3']).
 Funcall -> Identifier left_paren FunArg FunArgN right_paren : make_funcall('$1', ['$3'] ++ '$4').
 
+NullCond -> Vals NullComp : make_expr('$1', '$2').
```

Comment:
I suggest to rename "NullCond" to "NullPredicate" (as it is called in the SQL specs).